{
    "title": "An issue was discovered in the stack_dst crate before 0.6.1 for Rust. Because of the push_inner behavior, a double free can occur upon a val.clone() panic.",
    "score": 9.8,
    "severity": "CRITICAL",
    "cwe_id": "CWE-415",
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
    "cpe_match": [
        "cpe:2.3:a:stack_dst_project:stack_dst:*:*:*:*:*:rust:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1057",
            "confidence": "15.8",
            "techniqueName": "Process Discovery"
        },
        {
            "techniqueId": "T1087",
            "confidence": "13.4",
            "techniqueName": "Account Discovery"
        }
    ]
}
